src/librustc_driver/lib.rs

```diff
@@ -1460,6 +1460,8 @@ fn parse_crate_attrs<'a>(sess: &'a Session, input: &Input) -> PResult<'a, Vec<as
     }
 }
 
+const STACK_SIZE: usize = 4 * 1024 * 1024; // 4MB
```

Comment:
Maybe make this dependent on whether `stacker` *actually works*?

Reply:
How's that relevant? When stacker doesn't work, these values don't matter, because they don't do anything

Reply:
I thought this was the default thread stack size, unrelated to stacker. My bad if that's not the case.

Reply:
I'm gonna rename the constant to be clearer about this